docs/platforms/javascript/common/configuration/integrations/graphqlclient.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: GraphQL Client
+description: "Track healthy Sessions in the Browser."
+notSupported:
+  - javascript.cordova
+  - javascript.node
+  - javascript.aws-lambda
+  - javascript.azure-functions
+  - javascript.connect
+  - javascript.express
+  - javascript.fastify
+  - javascript.gcp-functions
+  - javascript.hapi
+  - javascript.hono
+  - javascript.koa
+  - javascript.nestjs
+  - javascript.deno
+  - javascript.cloudflare
+  - javascript.bun
+---
+
+<Alert>
+
+  This integration only works inside a browser environment.
+
+</Alert>
+
+_Import name: `Sentry.graphqlClientIntegration`_
+
+This integration enhances the data captured from GraphQL requests in your application.
+It extracts GraphQL-specific information from HTTP requests matching your configured endpoints and enriches
+both spans and breadcrumbs with GraphQL operation details.
+
+When enabled, this integration will:
+- Update span names with the GraphQL operation type and name
+- Add GraphQL query documents to spans
+- Add GraphQL operation information to breadcrumbs
+
+## Options
+
+### `endpoints`
+
+_Type: `(string|RegExp)[]`_
+
+An array of URLs or URL patterns that should be treated as GraphQL endpoints.
+The integration will only process requests to these endpoints. This array can contain strings, regular expressions, or a combination of both.
+
+Example:
+```javascript
+Sentry.init({
+  integrations: [
+    Sentry.graphqlClientIntegration({
+      endpoints: [
+        'https://api.example.com/graphql',
+        /\/graphql$/
```

Comment:
I understand the intention to illustrate both strings and regexes, but perhaps it is better to split this into two examples or to slightly modify this. In this case, the first entry would be redundant, as it would also match the second entry.

```suggestion
        'https://graphql-api.example.com',
        /\/graphql$/,
```

Reply:
Looks great now, thank you for the docs update!